### DISCUSSION TITLE
Impôt de guilde

### DISCUSSION THREAD

**Discussion by Iris-for** [Crownicles/Crownicles#1089] [Suggestions fermées]
Peut être créer une sorte d'impôts de guild et avec sa faire en sorte que le chef de la guild puisse améliorer la guilde et que si un chef / modo de guild sois inactif 14 jours qu’ils soient automatiquement remplacé

**Comment by BastLast** [comment]
je déplace cette suggestion dans les refusée car le remplacement de chef en cas de période d'innacitivté c'est super sensible et je pense pas que cela soit utile. 

Pr l'impot de guilde imo c'est à chacun de choisir comment la guilde s'organise
